### PR TITLE
feat: add U2U Nebulas Testnet

### PR DESCRIPTION
**Pull request by JunayedMh**

### CODE REVIEW

File: u2u-testnet/logo.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<svg width="1085" height="120" viewBox="0 0 1085 120" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (paulbalaji) [inline u2u-testnet/logo.svg:1]: please fix to include just the logo + run through an svg minifier  https://jakearchibald.github.io/svgomg/

File: u2u-testnet/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: " https://testnet.u2uscan.xyz/api"
```
Comment (paulbalaji) [inline u2u-testnet/metadata.yaml:3]:
```suggestion
  - apiUrl: https://testnet.u2uscan.xyz/api
```

File: u2u-testnet/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: " https://testnet.u2uscan.xyz/api"
+    family: blockscout
+    name: u2uscan
+    url: https://testnet.u2uscan.xyz/
```
Comment (paulbalaji) [inline u2u-testnet/metadata.yaml:6]:
```suggestion
    url: https://testnet.u2uscan.xyz
```

File: u2u-testnet/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+blockExplorers:
+  - apiUrl: " https://testnet.u2uscan.xyz/api"
+    family: blockscout
+    name: u2uscan
+    url: https://testnet.u2uscan.xyz/
+chainId: 2484
+displayName: U2U Testnet
+domainId: 2484
+isTestnet: true
+name: nebulatestnet
```
Comment (paulbalaji) [inline u2u-testnet/metadata.yaml:11]:
```suggestion
name: u2utestnet
```

File: u2u-testnet/metadata.yaml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+nativeToken:
+  decimals: 18
+  name: Union Ultra
+  symbol: U2U
+protocol: ethereum
+rpcUrls:
+  - http: https://rpc-nebulas-testnet.uniultra.xyz
```
Comment (paulbalaji) [inline u2u-testnet/metadata.yaml:18]:
```suggestion
rpcUrls:
  - http: https://rpc-nebulas-testnet.uniultra.xyz
```

**Review by paulbalaji — COMMENTED** [review]
rename `u2u-testnet` folder to `u2utestnet`

File: u2u-testnet/metadata.yaml (hunk 5 above)
Comment (JunayedMh) [reply u2u-testnet/metadata.yaml]: fixed all,kindly check again. thanks

**Comment by JunayedMh** [comment]
updated all files..waiting for review

File: u2utestnet/logo.svg [hunk 6]
```diff
@@ -0,0 +1 @@
...7h-9.609ZM937.957 106V69.613c0-4.336-.625-7.558-1.875-9.668-1.602-2.734-4.453-4.101-8.555-4.101-4.336 0-7.402 1.328-9.199 3.984-1.484 2.188-2.226 5.508-2.226 9.961V106H896.18V39.379h18.574v7.09h.293c3.828-6.25 9.863-9.375 18.105-9.375 6.758 0 12.442 1.933 17.051 5.8 2.891 2.422 4.961 5.684 6.211 9.786.977 3.203 1.465 7.09 1.465 11.66V106h-19.922ZM1038.5 73.48c0 1.914-.15 3.672-.46 5.274h-49.927c1.25 8.398 6.446 12.598 15.587 12.598 4.76 0 8.81-2.07 12.13-6.211h20.39c-2.97 6.914-7.23 12.421-12.77 16.523-5.9 4.336-12.49 6.504-19.75 6.504-9.727 0-18.028-3.457-24.903-10.371-6.914-6.875-10.371-15.176-10.371-24.903 0-9.765 3.379-18.144 10.136-25.136 6.836-7.11 15.098-10.664 24.788-10.664 10.08 0 18.49 3.554 25.25 10.664 6.6 6.992 9.9 15.566 9.9 25.723Zm-19.92-8.144c-1.64-7.617-6.68-11.426-15.12-11.426-8.433 0-13.472 3.809-15.112 11.426h30.232Zm54.9-10.84V106h-19.92V54.496h-9.84V39.38h9.84V17.172h19.92v22.207h9.61v15.117h-9.61Z"/><defs><linearGradient id="b" x1="53.091" x2="96.603" y1="60" y2="60" gradientUnits="userSpaceOnUse"><stop stop-color="#07806D"/><stop offset="1" stop-color="#035663"/></linearGradient><clipPath id="a"><path fill="#fff" d="M0 0h97v120H0z"/></clipPath></defs></svg>
```
Comment (paulbalaji) [inline u2utestnet/logo.svg:1]: done for you
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 120 120"><g clip-path="url(#a)"><path fill="#3C9" d="M44 120 0 95c4-1 4-4 4-6V31c0-3 0-5-4-6l19-11v74c0 3 1 5 5 7l4 3V12c0-2 1-4 3-5l13-7v120Z"/><path fill="url(#b)" d="m53 120 43-25c-3-1-3-4-3-6V31c0-3 0-5 4-6L77 14v74c0 3-1 5-4 7l-5 3V12c0-2 0-4-2-5L53 0v120Z"/><path fill="#fff" d="m61 8 1 4h4l-3 3 1 4-3-2-4 2 1-4-3-3h4l2-4Z"/></g><defs><linearGradient id="b" x1="53.1" x2="96.6" y1="60" y2="60" gradientUnits="userSpaceOnUse"><stop stop-color="#07806D"/><stop offset="1" stop-color="#035663"/></linearGradient><clipPath id="a"><path fill="#fff" d="M0 0h97v120H0z"/></clipPath></defs></svg>
```

File: u2utestnet/addresses.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+    staticAggregationHookFactory: "0x5050A9314BC0A90e3D6Ac3a9B43e0B8be4cB6029"
+    domainRoutingIsmFactory: "0x738fA638428964a2b2A73232F92c581124bA6f29"
+    staticMerkleRootWeightedMultisigIsmFactory: "0x602d791E966E7A672c79E5e1C31Bb4e35CcF4B36"
+    staticMessageIdWeightedMultisigIsmFactory: "0x56C3ea7bE6bC5Bc4B2B4f15e994a3Cd1B3ce56a2"
+    proxyAdmin: "0xd49B57A0Fd5E6e76715B3f1cefEf3Fd13E51a578"
+    mailbox: "0x97604104950f2c8e2F6314E7343Bc384d76c1e6f"
+    interchainAccountRouter: "0x8A8e3fD54ccAEb434640546ad5a5D8919E672Ddc"
+    interchainAccountIsm: "0x1Cb3b85770124a101bA6d8e99E0A782018cC6f7F"
+    validatorAnnounce: "0xb899155E5d8A43E571a65AA2ca9df85Cbd9bA7B0"
+    testRecipient: "0xA2a4563CbBC73dD3DD07aeDF8FEc76543F65cd4F"
```
Comment (paulbalaji) [inline u2utestnet/addresses.yaml:13]:
```suggestion
domainRoutingIsmFactory: "0x738fA638428964a2b2A73232F92c581124bA6f29"
interchainAccountIsm: "0x1Cb3b85770124a101bA6d8e99E0A782018cC6f7F"
interchainAccountRouter: "0x8A8e3fD54ccAEb434640546ad5a5D8919E672Ddc"
mailbox: "0x97604104950f2c8e2F6314E7343Bc384d76c1e6f"
proxyAdmin: "0xd49B57A0Fd5E6e76715B3f1cefEf3Fd13E51a578"
staticAggregationHookFactory: "0x5050A9314BC0A90e3D6Ac3a9B43e0B8be4cB6029"
staticAggregationIsmFactory: "0x3E39E57f5846722c7d48de8A1529156a1ef87110"
staticMerkleRootMultisigIsmFactory: "0xD705DD0cEA291b8b7f3D9856AEc48651A4B21fe0"
staticMerkleRootWeightedMultisigIsmFactory: "0x602d791E966E7A672c79E5e1C31Bb4e35CcF4B36"
staticMessageIdMultisigIsmFactory: "0xa3CbC64f3528B1baC7FC789B61535e8dBd6d35C0"
staticMessageIdWeightedMultisigIsmFactory: "0x56C3ea7bE6bC5Bc4B2B4f15e994a3Cd1B3ce56a2"
testRecipient: "0xA2a4563CbBC73dD3DD07aeDF8FEc76543F65cd4F"
validatorAnnounce: "0xb899155E5d8A43E571a65AA2ca9df85Cbd9bA7B0"
```

File: u2utestnet/addresses.yaml [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+mailbox: "0x97604104950f2c8e2F6314E7343Bc384d76c1e6f"
+proxyAdmin: "0xd49B57A0Fd5E6e76715B3f1cefEf3Fd13E51a578"
+staticAggregationHookFactory: "0x5050A9314BC0A90e3D6Ac3a9B43e0B8be4cB6029"
+staticAggregationIsmFactory: "0x3E39E57f5846722c7d48de8A1529156a1ef87110"
+staticMerkleRootMultisigIsmFactory: "0xD705DD0cEA291b8b7f3D9856AEc48651A4B21fe0"
+staticMerkleRootWeightedMultisigIsmFactory: "0x602d791E966E7A672c79E5e1C31Bb4e35CcF4B36"
+staticMessageIdMultisigIsmFactory: "0xa3CbC64f3528B1baC7FC789B61535e8dBd6d35C0"
+staticMessageIdWeightedMultisigIsmFactory: "0x56C3ea7bE6bC5Bc4B2B4f15e994a3Cd1B3ce56a2"
+testRecipient: "0xA2a4563CbBC73dD3DD07aeDF8FEc76543F65cd4F"
+validatorAnnounce: "0xb899155E5d8A43E571a65AA2ca9df85Cbd9bA7B0"
```
Comment (paulbalaji) [inline u2utestnet/addresses.yaml:13]:
```suggestion
domainRoutingIsmFactory: "0x738fA638428964a2b2A73232F92c581124bA6f29"
interchainAccountIsm: "0x1Cb3b85770124a101bA6d8e99E0A782018cC6f7F"
interchainAccountRouter: "0x8A8e3fD54ccAEb434640546ad5a5D8919E672Ddc"
mailbox: "0x97604104950f2c8e2F6314E7343Bc384d76c1e6f"
proxyAdmin: "0xd49B57A0Fd5E6e76715B3f1cefEf3Fd13E51a578"
staticAggregationHookFactory: "0x5050A9314BC0A90e3D6Ac3a9B43e0B8be4cB6029"
staticAggregationIsmFactory: "0x3E39E57f5846722c7d48de8A1529156a1ef87110"
staticMerkleRootMultisigIsmFactory: "0xD705DD0cEA291b8b7f3D9856AEc48651A4B21fe0"
staticMerkleRootWeightedMultisigIsmFactory: "0x602d791E966E7A672c79E5e1C31Bb4e35CcF4B36"
staticMessageIdMultisigIsmFactory: "0xa3CbC64f3528B1baC7FC789B61535e8dBd6d35C0"
staticMessageIdWeightedMultisigIsmFactory: "0x56C3ea7bE6bC5Bc4B2B4f15e994a3Cd1B3ce56a2"
testRecipient: "0xA2a4563CbBC73dD3DD07aeDF8FEc76543F65cd4F"
validatorAnnounce: "0xb899155E5d8A43E571a65AA2ca9df85Cbd9bA7B0"

```

File: u2utestnet/logo.svg [hunk 9]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 120 120"><g clip-path="url(#a)"><path fill="#3C9" d="M44 120 0 95c4-1 4-4 4-6V31c0-3 0-5-4-6l19-11v74c0 3 1 5 5 7l4 3V12c0-2 1-4 3-5l13-7v120Z"/><path fill="url(#b)" d="m53 120 43-25c-3-1-3-4-3-6V31c0-3 0-5 4-6L77 14v74c0 3-1 5-4 7l-5 3V12c0-2 0-4-2-5L53 0v120Z"/><path fill="#fff" d="m61 8 1 4h4l-3 3 1 4-3-2-4 2 1-4-3-3h4l2-4Z"/></g><defs><linearGradient id="b" x1="53.1" x2="96.6" y1="60" y2="60" gradientUnits="userSpaceOnUse"><stop stop-color="#07806D"/><stop offset="1" stop-color="#035663"/></linearGradient><clipPath id="a"><path fill="#fff" d="M0 0h97v120H0z"/></clipPath></defs></svg>
```
Comment (paulbalaji) [inline u2utestnet/logo.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 96 120"><g clip-path="url(#a)"><path fill="#3C9" d="M44 120 0 95c4-1 4-4 4-6V31c0-3 0-5-4-6l19-11v74c0 3 1 5 5 7l4 3V12c0-2 1-4 3-5l13-7v120Z"/><path fill="url(#b)" d="m53 120 43-25c-3-1-3-4-3-6V31c0-3 0-5 4-6L77 14v74c0 3-1 5-4 7l-5 3V12c0-2 0-4-2-5L53 0v120Z"/><path fill="#fff" d="m61 8 1 4h4l-3 3 1 4-3-2-4 2 1-4-3-3h4l2-4Z"/></g><defs><linearGradient id="b" x1="53.1" x2="96.6" y1="60" y2="60" gradientUnits="userSpaceOnUse"><stop stop-color="#07806D"/><stop offset="1" stop-color="#035663"/></linearGradient><clipPath id="a"><path fill="#fff" d="M0 0h97v120H0z"/></clipPath></defs></svg>
```

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**